src/components/PlantCareTipBlock/PlantCareTipBlock.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FunctionComponent, SVGAttributes, useEffect, useRef, useState } from 'react';
+import { Text, View } from 'react-native';
+import { useStyles } from 'react-native-unistyles';
+import { t } from 'i18next';
+import { styleSheet } from './PlantCareTipBlock.style';
+import { sizes } from '@src/styling/sizes';
```

Comment:
return object from `useStyles` has `theme` key. You can find sizes reference there